src/main/java/com/altenheim/kalender/controller/logicController/ExportController.java

```diff
@@ -0,0 +1,20 @@
+package com.altenheim.kalender.controller.logicController;
+
+import java.util.List;
+
+import com.altenheim.kalender.controller.viewController.CalendarViewOverride;
+import com.altenheim.kalender.interfaces.IAppointmentEntryFactory;
+import com.altenheim.kalender.interfaces.IExportController;
+import com.altenheim.kalender.models.ContactModel;
+import com.altenheim.kalender.models.SettingsModel;
+
+public class ExportController extends IOController implements IExportController
+{
+
```

Comment:
Müssen mal sehen ob das so hinhaut bei den Objektabhängigkeiten, erstmal leere Klasse in die Import Export ihre Methoden dann implementieren kann.